### PR TITLE
Replace kamaji-etcd with aenix-io/etcd-operator

### PR DESCRIPTION
**Pull request by kvaps**
This change replaces kamaji-etcd with our own [etcd-operator](https://github.com/aenix-io/etcd-operator/), which provides a robust environment for running etcd clusters.

The previous kamaji-etcd has not been removed yet, so it will continue to operate for compatibility purposes.

fixes: https://github.com/aenix-io/cozystack/issues/33

### CODE REVIEW

File: packages/system/etcd-operator/charts/etcd-operator/README.md
Comment (themoriarti) [inline packages/system/etcd-operator/charts/etcd-operator/README.md:27]: Why is the request CPU 100m? This is quite a lot, I think you should start with 20-40m, but you need to know how much etcd eats at least.

File: packages/system/etcd-operator/charts/etcd-operator/README.md
Comment (themoriarti) [inline packages/system/etcd-operator/charts/etcd-operator/README.md:26]: The memory limit is very small, an average k8s cluster with 100 pods will force etcd to use up to 300MB per instance. I think it should be increased to 384-512MB

File: packages/system/etcd-operator/charts/etcd-operator/README.md
Comment (themoriarti) [inline packages/system/etcd-operator/charts/etcd-operator/README.md:46]: This is quite a lot, I think you should start with 20-40m.

File: packages/system/etcd-operator/charts/etcd-operator/README.md
Comment (themoriarti) [inline packages/system/etcd-operator/charts/etcd-operator/README.md:45]: It's hard to say here, but it's worth reviewing the limit, you don't really want to catch OOM

**Comment by kvaps** [comment]
@themoriarti, there are defaults provided by the etcd-operator maintainers. We can consider other values, but I think it might be better to address this in the upstream project.

https://github.com/aenix-io/etcd-operator/

**Comment by themoriarti** [comment]
We are waiting for a response from etcd-operator https://github.com/aenix-io/etcd-operator/issues/199 , if they make changes it will simplify the integration.

File: packages/system/etcd-operator/charts/etcd-operator/values.yaml
Comment (themoriarti) [inline packages/system/etcd-operator/charts/etcd-operator/values.yaml:30]: need more memory up to 512MB

File: packages/system/etcd-operator/charts/etcd-operator/values.yaml
Comment (themoriarti) [inline packages/system/etcd-operator/charts/etcd-operator/values.yaml:32]: for start etcd not need more then 40m CPU reservation

**Comment by themoriarti** [comment]
@kvaps Can you add the default value of requests and resource limits when creating an etсd cluster?

**Review by themoriarti — APPROVED** [review]
At the moment, we have gone through all the discussions and have come to the decision not to set any additional limits, but after the release we will monitor and return to the discussion of defaults in the next discussions. Fly fly our good etсd.

**Pull request merged**
